packages/fuels-core/src/traits/tokenizable.rs

```diff
@@ -557,6 +577,27 @@ mod tests {
         assert_eq!(token, Token::B256(data));
     }
 
+    #[test]
+    fn test_from_token_raw_slice() -> Result<(), Error> {
+        let data = vec![42; 11];
+        let token = Token::RawSlice(data.clone());
+
+        let slice = RawSlice::from_token(token)?;
+
+        assert_eq!(slice.0, data);
```

Comment:
You implemented `PartialEq` for `Vec`
```suggestion
        assert_eq!(slice, data);
```